### ISSUE TITLE
[Bug] Setting "see what's new..." cannot be unchecked

### ISSUE THREAD

**Issue by Linnet5** [ungive/discord-music-presence#50]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Setting "see what's new when launching a new version" cannot be unchecked, instead unchecks "notify when a new version is available".

### Reproduction Steps

1. Settings
2. Click on see what's new when launching a new version.
3. Notify when a new version is available gets unchecked.

### Expected behavior

"See what's new when launching a new version" is expected to behave like the other checkboxes.

### Device Info

- Discord Music Presence version: [e.g. 2.2.3]
- Operating system: [e.g. Windows]
- The media player you used (if applicable): Apple Music

**Comment by ungive** [comment]
Good catch! I can't believe I never actually clicked that checkbox.

As a workaround you can go to `%APPDATA%\Music Presence\settings.json` on Windows and `~/Library/Application Support/Music Presence/settings.json` on Mac (off the top of my head) and edit these lines:

```json
  "miscellaneous": {
    "show_update_popups": true,
    "show_first_launch_changelog": true
  },
```

`show_update_popups` corresponds to `Notify when a new version is available`.

`show_first_launch_changelog` corresponds to `See what's new when launching a new version`.

Change `true` to `false` to disable either of them, if you like.

This will of course be fixed in the next version, which will arrive soon. Thanks for reporting!

**Comment by ungive** [comment]
Fixed in version 2.2.4
